photon-api/src/main/scala/com/linkedin/photon/ml/ModelTraining.scala

```diff
@@ -112,11 +106,10 @@ object ModelTraining extends Logging {
       normalizationContext: NormalizationContext,
       maxNumIter: Int,
       tolerance: Double,
-      enableOptimizationStateTracker: Boolean,
       constraintMap: Option[Map[Int, (Double, Double)]],
       warmStartModels: Map[Double, GeneralizedLinearModel],
       treeAggregateDepth: Int,
-      useWarmStart: Boolean): List[(Double, _ <: GeneralizedLinearModel, Option[OptimizationStatesTracker])] = {
+      useWarmStart: Boolean): List[(Double, GeneralizedLinearModel, OptimizationStatesTracker)] = {
```

Comment:
I was getting a casting error during testing, and this resolved it. Overall, it was an unnecessary type anyway since the explicit type of the models (e.g. logistic, linear, Poisson) is never checked/used and the models are cast to `GeneralizedLinearModel`.